content/en/docs/started/support.md

```diff
@@ -4,224 +4,99 @@ description = "Where to get support for Kubeflow"
-  the application stable for this release of Kubeflow.
-* **Beta** means that the application is working towards a version 1.0 release
-  and its maintainers have communicated a timeline for satisfying the criteria
-  for the stable status.
-* **Alpha** means that the application is in the early phases of
-  development and/or integration into Kubeflow.
+## Component Status
+
+When you deploy Kubeflow to a Kubernetes cluster, your deployment includes multiple components. Note that component versioning is independent from Kubeflow versioning. Each component should meet certain [criteria](https://github.com/kubeflow/community/blob/master/guidelines/application_requirements.md) regarding stability, upgradability, logging, monitoring, and security (PodSecurityStandards restricted, network policies, and integration tests for authentication and authorization).
+
+Component status indicators:
+
+* **Stable**: The application complies with most of the criteria and is considered stable for this release.
+* **Beta**: The application is progressing towards meeting most criteria.
+* **Alpha**: The application is in an early development or integration stage.
 
 <a id="levels-of-support"></a>
-## Levels of support
-
-The following table describes the level of support that you can expect based on the status of an application:
-
-<div class="table-responsive">
-  <table class="table table-bordered">
-    <thead class="thead-light">
-      <tr>
-        <th>Application status</th>
-        <th>Level of support</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <td>Stable</td>
-        <td>The Kubeflow community offers <i>best-effort support</i> for stable
-          applications. See the section on 
-          <a href="#community-support">community support</a> below for a
-          definition of best-effort support and the community channels where you 
-          can report and discuss the problem. You can also consider requesting 
-          support from a 
-          <a href="#provider-support">Kubeflow community provider</a> or from 
-          your <a href="#cloud-support">cloud provider</a>.
-        </td>
-      </tr>
-      <tr>
-        <td>Beta</td>
-        <td>The Kubeflow community offers <i>best-effort support</i> for beta
-          applications. See the section on 
-          <a href="#community-support">community support</a> below for a
-          definition of best-effort support and the community channels where you 
-          can report and discuss the problem. 
-        </td>
-      </tr>
-      <tr>
-        <td>Alpha</td>
-        <td>The response differs per application in alpha status, depending on
-          the size of the community for that application and the current level
-          of active development of the application.</td>
-      </tr>
-    </tbody>
-  </table>
-</div>
+## Levels of Support
+
+1. The Kubeflow community offers best-effort support for stable components.
```

Comment:
```suggestion
1. The Kubeflow community provides best-effort support for stable components, with no guarantees, as it is free and community-maintained software.
```

Reply:
Maybe this is more explicit?

Reply:
I think best efort is explained further down, but we can replace "offers" with "provides"

Reply:
solved